nautobot/wireless/filters.py

```diff
@@ -22,6 +28,10 @@ class AccessPointGroupFilterSet(NautobotFilterSet, TenancyModelFilterSetMixin):
         to_field_name="name",
         label="Controller (name or ID)",
     )
+    has_devices = RelatedMembershipBooleanFilter(
+        field_name="devices",
+        label="Has devices",
+    )
```

Comment:
No corresponding `devices` filter?

Reply:
I will add, I missed when changing this from using a M2M( with through table) to being an FK on the Device model.